app/models/court.rb

```diff
@@ -0,0 +1,5 @@
+class Court < ActiveRecord::Base
+  after_validation :geocode, if: ->(obj){ obj.address.present? and obj.address_changed? }
```

Comment:
Line is too long. [89/80]<br>Use `&&` instead of `and`.<br>Space missing to the left of {.